src/toolbox/ngm-point-edit.js

```diff
@@ -82,6 +88,14 @@ class NgmPointPosition extends I18nMixin(LitElement) {
     this.viewer.scene.requestRender();
   }
 
+  onColorChange(color) {
+    this.entity.billboard.color = color;
+  }
+
+  onSymbolChange(image) {
+    this.entity.billboard.image = `../images/${image}`;
```

Comment:
See above, shouldn't it be? :
```suggestion
    this.entity.billboard.image = `./images/${image}`;
```